### ISSUE TITLE
Unable to start

### ISSUE THREAD

**Issue by tkna91** [JoepVanlier/Hackey-Trackey#105]
I seem to be unable to start the program with the following error.
What should I do?
```lua
tracker.lua:4216: attempt to perform arithmetic on a nil value (field 'windowHeight')
```
Screencast: 

https://github.com/JoepVanlier/Hackey-Trackey/assets/102382754/cc8cbbd9-1491-4696-836f-3f5898aa22e6

Environment:
- Hackey-Trackey v3.30
- reaper 7.16-1
- OS: Arch Linux x86_64
- Kernel: 6.9.2-arch1-1
- WM: sway

**Comment by JoepVanlier** [comment]
Yep, that's a pretty bad bug. This thing really needs a complete rewrite with stuff learned over the years, but lack of time at the moment.

Anyways, thank you for reporting it. It should be fixed in `3.31` which I just pushed. Please don't hesitate to report further issues. I still do my best to fix reported bugs.

**Comment by tkna91** [comment]
I was able to start the program. Thanks!
![image](https://github.com/JoepVanlier/Hackey-Trackey/assets/102382754/02628ea6-d2cc-42a2-ae33-b3dfeb87a2d6)
